.github/workflows/code_review_infra.yaml

```diff
@@ -0,0 +1,27 @@
+name: Code Review Infra
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - edited
```

Comment:
```suggestion
```

According to [the official documentation](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request)
> The title or body of a pull request was edited, or the base branch of a pull request was changed.